platform-hub-api/app/controllers/kubernetes/clusters_controller.rb

```diff
@@ -116,7 +116,8 @@ def cluster_params
       :s3_secret_access_key,
       :s3_object_key,
       :api_url,
-      :ca_cert_encoded
+      :ca_cert_encoded,
+      aliases: [],
```

Comment:
Drop the comma on the end? Consistent with other definitions

Reply:
Yeah this is one of those weird things Ruby allows that I'm never consistent with 😄 On the one hand it's useful to keep in since future updates to the list will result in less lines of diff. On the other hand… it doesn't matter as much :) Will update to keep consistent, at least.